config/install/migrate_plus.migration.y_lb_demo_events.yml

```diff
@@ -34,9 +34,9 @@ source:
       title: 'Event Title'
       body: "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>"
       summary: "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
-      contact_email: sXyimJuV@example.com
-      contact_name: 'John Doe'
-      contact_phone: '+1 (888) 123-4567'
+      contact_email: events@ymcaspringfield.org
```

Comment:
There is no such domain
We need to keep example.org or register it in order not confuse end users or do not create false link if anyone would decide to play fraud and register domain to hijack traffic